portality/lib/coerce.py

```diff
@@ -73,7 +73,8 @@ def to_country_code(val):
     return uc(nv)
 
 def to_issn(issn):
-    if len(issn) > 9:
+
+    if len(issn) > 9 or issn == '':
```

Comment:
Can we look upstream to whichever crosswalks are writing issns into the data, and make sure that they are now checking for empty issns, and just omitting them?  That will prevent error cases in most normal "user error" kind of ways, while leaving us a defence here for anywhere they slip through the net.  I'd guess the journal form crosswalk is the place to do it?

Reply:
I tested the code and now if the empty string is provided as issn it is just omitted. I will add some UT to make sure it works fine  :)

Reply:
It looks to me that `allow_coerce_failure` is set on the `IncomingApplication` data object, which means if there's an exception here, that will be swallowed and the empty string used as the issn value.  This implies that the empty string would be entered into the issn field, and I can't see anywhere that it checks for the empty string, so I'd want to double check that it isn't writing that empty string into the issn field.